src/main/java/club/sk1er/patcher/mixins/plugin/PatcherMixinPlugin.java

```diff
@@ -52,7 +53,11 @@ public void acceptTargets(Set<String> myTargets, Set<String> otherTargets) {
 
     @Override
     public List<String> getMixins() {
-        return null;
+        //#if MC==10809
+        return Lists.newArrayList("bugfixes.optifine.RegionRenderCacheMixin_FixConnectedTextures");
```

Comment:
Unnecessary, our MixinPlugin stops anything in a `.optifine` package from being applied if OptiFine isn't found.